lib/createWorker.js

```diff
@@ -18,7 +13,11 @@ const {
  */
 function formatCreateWorkersParameters(handler, config, options) {
   return {
-    handlers: [{ handle: handler, validate: options.validator, routingKey: config.routingKey }],
+    handlers: [{
+      handle: handler,
+      validate: options.validator || _.identity,
```

Comment:
Please remove the default value. Schema validation must absolutely be required for AMQP handlers, like it is for HTTP controllers